src/main/java/entities/validity_checkers/WordValidityCheckerRegular.java

```diff
@@ -0,0 +1,27 @@
+package entities.validity_checkers;
+
+/**
+ * Word validity checker for the regular game mode
+ */
+public class WordValidityCheckerRegular implements WordValidityChecker{
+
+    private static final int WORD_LENGTH = 22;
+
+    /**
+     * Verifies word is valid based on game mode regular criteria
+     * Makes changes to the word if error is forgivable
+     * @param word the word to validate (trimmed)
+     * @return valid word, or if not valid, null
+     */
+    @Override
+    public String isWordValid(String word) {
```

Comment:
It is probably a good idea to have a minimum length. an empty string should not be a valid word